src/kOS/Communication/StockConnectivityManager.cs

```diff
@@ -1,14 +1,50 @@
 ﻿using System;
+using CommNet;
+using kOS.Module;
 
 namespace kOS.Communication
 {
-    public class StockConnectivityManager : ConnectivityManager
+    public class StockConnectivityManager : IConnectivityManager
```

Comment:
Maybe `IgnoreConnectivityManager`?  I kind of thought about changing it, but couldn't come up with a good name.  So I left it for others to comment on.  I'm in favor of changing the name, I'm just not sure what a good choice would be.